autoscheduler/frontend/src/tests/ui/Availability.test.tsx

```diff
@@ -276,6 +346,570 @@ describe('Availability UI', () => {
       expect(queryByLabelText('Adjust End Time'))
         .toHaveAttribute('aria-valuetext', expectedEnd);
     });
+
```

Comment:
This is almost 600 lines of tests for multi-day availabilities - I think it's worth making this it's own file